.npmignore

```diff
@@ -0,0 +1,8 @@
+NodeReport*.txt
```

Comment:
What if you blacklist everything by default and then add things back. Something like this could work:

```
*
!src/**
!binding.gyp
```